src/main/java/com/cburch/logisim/analyze/model/Assignments.java

```diff
@@ -34,8 +34,6 @@
 class Assignments {
   private final Map<String, Boolean> map = new HashMap<>();
 
-  public Assignments() {}
```

Comment:
Why you remove this constructor and left similar ones in place as dummy in other places?

Reply:
This one is `public` and the only constructor, so can be removed. The other places I marked with `// dummy` were private constructors.